common/djangoapps/third_party_auth/models.py

```diff
@@ -366,13 +366,12 @@ class OAuth2ProviderConfig(ProviderConfig):
 
     .. no_pii:
     """
-    # We are keying the provider config by backend_name here as suggested in the python social
```

Comment:
I guess we didn't provide a link anyway, but we don't think it is useful to point note python social auth documentation any more? Is that because we are only partially following the recommendation? Or just that you think it isn't helpful at all?

Reply:
I could not find any documentation about provider config in python social auth and it doesn't seem to be related to it.

Reply:
🤷